lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -34,16 +37,38 @@
   private final int megaBytes;
   private final int number;
   private final int cores;
+  @Deprecated
```

Comment:
Please add documentation as to when it was deprecated (0.12) and what to use instead.